server/impl/src/main/java/com/walmartlabs/concord/server/org/project/ProjectValidator.java

```diff
@@ -121,7 +121,7 @@ private static void validateSpec(Trigger t, List<String> errors) {
 
     private static void validateTimezone(Trigger t, List<String> errors) {
         String triggerName = t.name();
-        if (!triggerName.equals("cron")) {
+        if (!"cron".equals(triggerName)) {
```

Comment:
Maybe useful?

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behavior is otherwise equivalent, there is no cost to this change.